doc/classes/EditorDock.xml

```diff
@@ -87,14 +93,19 @@
 		<member name="title" type="String" setter="set_title" getter="get_title" default="&quot;&quot;">
 			The title of the dock's tab. If empty, the dock's [member Node.name] will be used. If the name is auto-generated (contains [code]@[/code]), the first child's name will be used instead.
 		</member>
+		<member name="title_color" type="Color" setter="set_title_color" getter="get_title_color" default="Color(0, 0, 0, 0)">
+			Color of the dock tab's title. If transparent, default color will be used.
```

Comment:
Misleading. What does "transparent" mean here? Fully transparent? Even just a bit transparent? Or is it only fully transparent black? And what is the "default" color?